tools/clang/test/CodeGenSPIRV/hull.pcf.input-patch.hlsl

```diff
@@ -0,0 +1,44 @@
+// Run: %dxc -T hs_6_0 -E main
+
+#include "bezier_common_hull.hlsl"
+
+// Test: PCF takes the input control points (InputPatch)
+
+// CHECK: OpEntryPoint TessellationControl %main "main" %in_var_hullEntryPointInput {{%\w+}}
+
+// CHECK:              [[fType:%\d+]] = OpTypeFunction %HS_CONSTANT_DATA_OUTPUT %_ptr_Function__arr_VS_CONTROL_POINT_OUTPUT_uint_16
+// CHECK: %in_var_hullEntryPointInput = OpVariable %_ptr_Input__arr_VS_CONTROL_POINT_OUTPUT_uint_16 Input
+// CHECK:               %param_var_ip = OpVariable %_ptr_Function__arr_VS_CONTROL_POINT_OUTPUT_uint_16 Function
```

Comment:
I'd suggest to add `// CHECK-LABEL: %main = OpFunction` or some comments to make it clear the following is in the wrapper.